apps/persons/tests/utils.py

```diff
@@ -0,0 +1,124 @@
+"""Test utils for the courses application"""
+
+from django.contrib.auth.models import AnonymousUser, Permission
+from django.core.exceptions import ImproperlyConfigured
+from django.test.client import RequestFactory
+
+from cms.toolbar.items import Menu, ModalItem
+from cms.middleware.toolbar import ToolbarMiddleware
+
+from apps.core.factories import UserFactory
+
+
+class CheckToolbarMixin:
+    """
+    Mixin that factorize `check_toolbar_item` method used by several applications
```

Comment:
used TO TEST several applications

Reply:
I think the whole phrase is right, don't you ?

"""Mixin that factorize `check_toolbar_item` method used by several applications CMS toolbar unit tests"""

Reply:
:+1: